src/components/Sheet/getSheet.ts

```diff
@@ -184,6 +190,10 @@ export const getSheet = (
           item.graphicUri = info[apmDimStr].content;
         }
       }
+      const titleMediaId = related(section, 'titleMediafile');
+      item.titleMediaId = titleMediaId
+        ? { type: 'mediafile', id: titleMediaId }
```

Comment:
I agree it would be simpler. I was debating doing it that way. Then I looked at model/sheet.tsx and this is the schema for ISheet:
export interface ISheet {
  level: SheetLevel; //currently not used anywhere
  kind: IwsKind;
  sectionSeq: number;
  title?: string;
  transcriber?: RecordIdentity;
  editor?: RecordIdentity;
  sectionId?: RecordIdentity;
  sectionUpdated?: string;
  passageSeq: number;
  book?: string;
  reference?: string;
  comment?: string;
  passage?: Passage;
  sharedResourceId?: RecordIdentity;
  passageType: PassageTypeEnum;
  passageUpdated?: string;
  mediaId?: RecordIdentity;
  mediaShared: IMediaShare;
  step?: string;
  stepId?: string;
  deleted: boolean;
  filtered: boolean;
  discussionCount: number;
  published: boolean;
  graphicUri?: string;
  titleMediaId?: RecordIdentity;
}

It looked like all the other fields that were called Id in this structure used RecordIdentity so that is the way I coded it. I could certainly change it. We probably need a naming convention for when an Id is a RecordIdentity and when it is just the string for the id part of it.